docs/repositories-configure/configuring-code-patterns.md

```diff
@@ -4,9 +4,9 @@ description: Configure the tools and code patterns that Codacy uses to analyze y
 
 # Configuring code patterns
 
-By default, Codacy uses a subset of the code patterns that exist for each tool to analyze your repositories. These default code patterns result from community feedback or existing coding standards.
+By default, Codacy uses a subset of the code patterns that exist for each tool to analyze your repositories. These default settings result from community feedback or existing coding standards.
```

Comment:
Any reason that we are changing this wording here?

Reply:
While working on these changes I realized that we not only had code patterns enabled by default but tools as well.

The previous sentence only mentioned the code patterns, and by changing "code patterns" to "settings" the sentence becomes more generic and can apply to the tools well.

But even so, now that I'm looking at this again, the fact that we have only some tools enabled by default can be made more explicit. So I'm going to give it another go! :+1:

Reply:
@machadoit, you can check an updated version here: https://github.com/codacy/docs/pull/823/files#diff-8b5b34d427af25055560dd40e0c14ca5d6e7e029739f9870df33f810d6435ee1